packages/database/supabase/migrations/20240223235518_add_search_functionality_to_equipment_and_work_cell_types.sql

```diff
@@ -0,0 +1,85 @@
+
+
+-- Work Cell Type Start
+CREATE FUNCTION public.create_work_cell_type_search_result()
+RETURNS TRIGGER AS $$
+BEGIN
+  INSERT INTO public.search(name, description, entity, uuid, link)
+  VALUES (new.id, new.id || ' ' || new.name || ' ' || COALESCE(new.description, ''), 'Part', new.id, '/x/resources/work-cells/' || new.id);
```

Comment:
The name is what's returned in the search result. Let's use the name for the name (instead of the id).

![Screenshot 2024-02-26 at 10 33 08 PM](https://github.com/barbinbrad/carbon/assets/64510427/fdc5f12c-9c4b-429f-8fad-912f00046d5a)


```suggestion
  VALUES (new.name, COALESCE(new.description, ''), 'Resource', new.id, '/x/resources/work-cells/' || new.id);
```